common-lib/src/main/scala/com/gu/mediaservice/lib/elasticsearch/Mappings.scala

```diff
@@ -75,7 +75,8 @@ object Mappings {
     "subLocation" -> standardAnalysedString,
     "city" -> standardAnalysedString,
     "state" -> standardAnalysedString,
-    "country" -> standardAnalysedString
+    "country" -> standardAnalysedString,
+    "sStemmerCatchAll" -> sStemmerAnalysedString
```

Comment:
Not sure sStemmer is the right name as it's having a few things happen to it. 
[There's a comment here about it](https://github.com/guardian/grid/blob/master/common-lib/src/main/scala/com/gu/mediaservice/lib/elasticsearch/IndexSettings.scala#L14).

Could be called `englishAnalysedCatchAll`?